docs/misc/deprecated.md

```diff
@@ -12,6 +12,14 @@ parent = "mn_use_docker"
 
 The following list of features are deprecated.
 
+### HostConfig at API container start
+**Deprecated In Release: v1.9**
+
+**Target For Removal In Release: v1.11**
+
+Passing an `HostConfig` to `/container/{name}/start` is deprecated in favor of
+defining it at container creation or run.
```

Comment:
since this is describing a deprecation in the API, I don't think we need to mention `run`, as that is a CLI command (which does create+start). Perhaps better to mention the "create" *endpoint* instead?